test/contracts/ConsensusMock.sol

```diff
@@ -39,6 +39,10 @@ contract ConsensusMock is Consensus {
     boolStorage[SHOULD_EMIT_INITIATE_CHANGE] = _status;
   }
 
+  function getMaxValidators() public pure returns(uint256) {
+    return 3;
```

Comment:
did you test it with 100 as it should be?

Reply:
Just manually the function itself, but I will add a test